src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/_help.py

```diff
@@ -568,3 +577,27 @@
         - name:  Disable Service Bus Migration of Standard to Premium namespace
           text: az servicebus migration abort --resource-group myresourcegroup --name standardnamespace
 """
+
+helps['servicebus namespace network-rule add'] = """
+    type: command
+    short-summary: Set a Network rule for a Namespace
+    examples:
+        - name: set a Network rule for a namespace
+          text: az servicebus namespace network-rule add --resource-group myresourcegroup --namespace-name mynamespace
+"""
+
+helps['servicebus namespace network-rule list'] = """
+    type: command
+    short-summary: Show properties of Network rule of the given Namespace
+    examples:
+        - name: Show properties of Network rule of the given Namespace
+          text: az servicebus namespace network-rule list --resource-group myresourcegroup --namespace-name mynamespace
+"""
+
+helps['servicebus namespace network-rule remove'] = """
+    type: command
+    short-summary: Remove Network Rule of the given Namespace
+    examples:
+        - name: Remove Network rulet of the given Namespace
```

Comment:
Typo. "Remove a network rule from the namespace."

Reply:
resolved